src/java.base/share/classes/java/lang/Identity.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Reply:
(This has been discussed in mainline before. If both years are equal, there should only be one)